src/util/CMakeLists.txt

```diff
@@ -36,11 +27,32 @@ add_library(bitcoin_util STATIC EXCLUDE_FROM_ALL
   ../sync.cpp
 )
 
-target_link_libraries(bitcoin_util
+target_link_libraries(bitcoin_kernel_util
   PRIVATE
     core_interface
     bitcoin_clientversion
```

Comment:
Should also be easy to get `bitcoin_clientversion` out of here.

Reply:
I think it is just used for logging. I could just stub it out, like we do for the translation function, but I am also not as sure how important it really is to be printing the clientversion there for these logs.

Reply:
Yea, we could probably drop that. There's also `CLIENT_VERSION` fed into the env RNG, which could also likely be dropped.